### ISSUE TITLE
[Feature Request] Expose config for excluding gen files

### ISSUE THREAD

**Issue by vintage** [FlutterGen/flutter_gen#60]
I've been generating the assets and got surprised by unexpected records. Well, its kinda expected (the file exists there), but would be great to ignore these, as I don't treat `.DS_Store` as my project files :)

```
...
String get dSStore => 'animations/.DS_Store';
...
```

Any chance to expose `exclude` config (similar to the one we have in `analasis_options.yaml`) or at least ignore some well known files by default? <3

**Comment by wasabeef** [comment]
@vintage 
Thank you for your report.
 
We just released 1.3.1 please use it. 🎯 
```shell
$ brew upgrade fluttergen/tap/fluttergen
```

```shell
$ dart pub global activate flutter_gen
```

**Comment by vintage** [comment]
@wasabeef Wow, thank you for the instant fix! Upgraded to `1.3.1` and the problem is gone :)
